src/renderers/server-render.jsx

```diff
@@ -94,7 +93,10 @@ const getRouterRenderer = (store, baseUrl, clientFilename, assetPublicPath) =>
 				200;
 		} catch(e) {
 			// log the error stack here because Observable logs not great
-			console.error(e.stack);
+			console.error(JSON.stringify({
+				message: `Internal rendering error ${e.message}`,
```

Comment:
this is just a vanilla `try/catch` JS Error https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/prototype

I'll have another look at refactoring `getRouterRenderer` to consume the original Hapi `request` if possible